pkiintergration-fe/src/modules/pki-managed-certificates/pki-managed-certificates.component.ts

```diff
@@ -2,19 +2,23 @@ import { Component, OnInit } from '@angular/core';
 import { BsModalService } from 'ngx-bootstrap/modal';
 import { FetchClient } from '@c8y/client';
 import { CreateCertificateComponent } from './create-certificate/create-certificate.component';
+import { AlertService } from '@c8y/ngx-components';
+import { CreateCertificateService } from '@services/create-certificate.service';
 @Component({
   selector: 'app-pki-managed-certificates',
   templateUrl: './pki-managed-certificates.component.html'
 })
 export class PkiManagedCertificatesComponent {
   data: any;
 
-  constructor(private fetchClient: FetchClient, private modalService: BsModalService) {}
+  constructor(
+    private fetchClient: FetchClient, 
+    private modalService: BsModalService,
+    private alertService: AlertService,
+    private certificateService: CreateCertificateService,
+    ) {}
 
-  async ngOnInit() {
-    const response = await this.fetchClient.fetch('/certificates');
-    this.data = await response.json();
-  }
+ 
 
   createCertificate() {
     this.modalService.show(CreateCertificateComponent, {
```

Comment:
(line 27)
todo: use appropriate variable declaration

sub is not re-declared anywhere; use const